WORKSPACE

```diff
@@ -37,10 +37,10 @@ bind(
 # 1. Determine SHA256 `wget https://github.com/envoyproxy/envoy-wasm/archive/$COMMIT.tar.gz && sha256sum $COMMIT.tar.gz`
 # 2. Update .bazelversion, envoy.bazelrc and .bazelrc if needed.
 #
-# Commit time: 6/30/20
-ENVOY_SHA = "06d236f62097bebc3af7b4f3154b7beca73f1339"
+# Commit time: 7/13/20
```

Comment:
nit: This should be `Commit time` or `Commit date`? Because there is no time mentioned on the date.

Reply:
Done.